ckanext/iati_generator/templates/package/iati_page.html

```diff
@@ -0,0 +1,8 @@
+{% ckan_extends %}
```

Comment:
This paga is not, it's not extending from any CKAN core template.
Considering this is a new tab in the package read page here we need something like this

```suggestion
{% extends "package/read_base.html" %}
```

Reply:
https://github.com/okfn/ckanext-iati-generator/pull/6/commits/108ef9b272d169c9417795f4298fef5e6e579428